src/test/scala/org/broadinstitute/dsde/firecloud/service/FireCloudDirectivesSpec.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.scalatest.FreeSpec
+import spray.testkit.ScalatestRouteTest
+
+class FireCloudDirectivesSpec extends FreeSpec with ScalatestRouteTest with FireCloudDirectives {
+
+  def actorRefFactory = system
+
+  "FireCloudDirectives" - {
+    "Unencoded passthrough URLs" - {
+      "should be encoded" in {
+        val unencoded = "http://abc.com/path with spaces/"
+        val encoded = encodeUri(unencoded)
+        println(encoded)
```

Comment:
Can we add an assertion here instead of just `println`s? The example I sent you in hipchat was meant as a starter - sorry if I wasn't more clear on that.